docs/csharp/language-reference/builtin-types/floating-point-numeric-types.md

```diff
@@ -0,0 +1,99 @@
+
+The `decimal` type has more precision and a smaller range than both `float` and `double`, which makes it appropriate for financial and monetary calculations.
+
+You can mix numeric integral types and floating-point types in an expression. In this case, the integral types are converted to floating-point types. The evaluation of the expression is performed according to the following rules:
+
+- If one of the floating-point types is `double`, the expression evaluates to `double`, or to [bool](../keywords/bool.md) in relational comparisons or comparisons for equality.
+- If there is no `double` type in the expression, the expression evaluates to `float`, or to [bool](../keywords/bool.md) in relational comparisons or comparisons for equality.
+
+A floating-point expression can contain the following sets of values:
+
+- Positive and negative zero
+- Positive and negative infinity
+- Not-a-Number value (NaN)
+- The finite set of nonzero values
+
+For more information about these values, see IEEE Standard for Binary Floating-Point Arithmetic, available on the [IEEE](https://www.ieee.org) website.
+
+## floating point literals
+
+By default, a real numeric literal on the right side of the assignment operator is treated as `double`. You can use suffixes to convert a literal to a specific type. 
+
+- The `d` or `D` suffix converts an integral literal to a `double`.
+- The `f` or `F` suffix converts a real or integral literal to a `float`.
+- The `m` or `M` suffix converts a real or integral literal to a `decimal`.
+
+The following examples show each suffix:
+
+```csharp
+double d = 3D;
+d = 4d;
+float f = 3.5F;
+f = 5.4f;
+decimal myMoney = 300.5m;
+myMoney = 400.75M;
+```
+
+## Conversions
+
+There's an implicit conversion (called a *widening conversion*) between any two floating-point types where the destination type can store all values of the source type. For example, there's an implicit conversion from `float` to `double` because the range of `float` values is a proper subset of `double`. 
+
+You must use an explicit cast to convert one floating-point type to another floating-point type when an implicit conversion isn't defined from the source type to the destination type. This is called a *narrowing conversion*. The explicit case is required because the conversion can result in data loss. There's no implicit conversion between other floating-point types and the `decimal` type because the `decimal` type has greater precision than either `float` or `double`.
+
+For more information about implicit numeric conversions, see [Implicit Numeric Conversions Table](../keywords/implicit-numeric-conversions-table.md).
+
+For more information about explicit numeric conversions, see [Explicit Numeric Conversions Table](../keywords/explicit-numeric-conversions-table.md).
+
+## Formatting decimal output
```

Comment:
Is it really specific to `decimal`? Or intention is "Formatting floating-point values"
Is it worth to mention string interpolation instead of `Console.Write`?